packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/PushNotificationUtils.kt

```diff
@@ -0,0 +1,116 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.amazonaws.amplify.amplify_push_notifications
+
+import android.content.Intent
+import android.os.Bundle
+import com.amplifyframework.pushnotifications.pinpoint.utils.NotificationPayload
+import com.amplifyframework.pushnotifications.pinpoint.utils.PushNotificationsConstants
+import com.google.firebase.messaging.RemoteMessage
+
+// TODO: Revisit this file and remove un-used functions
+
+fun getBundleFromRemoteMessage(remoteMessage: RemoteMessage): Bundle {
```

Comment:
Use extension methods to organize this code:

```suggestion
fun RemoteMessage.asBundle(): Bundle {
```